FEXCore/Source/Interface/Core/LookupCache.h

```diff
@@ -89,8 +89,8 @@ class LookupCache {
   void AddBlockMapping(uint64_t Address, void* HostCode) {
     std::lock_guard<std::recursive_mutex> lk(WriteLock);
 
-    [[maybe_unused]] auto Inserted = BlockList.emplace(Address, (uintptr_t)HostCode).second;
-    LOGMAN_THROW_A_FMT(Inserted, "Duplicate block mapping added");
+    // This may replace an existing mapping
+    BlockList.emplace(Address, (uintptr_t)HostCode);
```

Comment:
Judging by the comment, is the intent here to *override* the previous HostCode value? `emplace` won't do this, so you have to use `operator[]` instead.

Reply:
Yep - thanks for spotting